src/components/SnazzyButton.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+
+
+const styles = {
+  snazzyButton: {
```

Comment:
This style you just made up? 
You need to import the style from the helper.ts like we were talking about yesterday.